### PR TITLE
Added an Example for ImageStream

### PR DESCRIPTION
**Pull request by piyush1594**
#1046

### CODE REVIEW

**Comment by oscerd** [comment]
ok to test

**Comment by piyush1594** [comment]
@oscerd Please do not merge it for the time being.

**Comment by oscerd** [comment]
Sure, WIP label is explicit enough :-)

File: kubernetes-examples/src/main/java/io/fabric8/openshift/examples/ImageStreamExample.java [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+      log("Created ImageStream", client.imageStreams().inNamespace(namespace).create(imageStream));
+
+    }finally {
+
+      log("Tags in ImageStream are");
+      log("First Tag is " + client.imageStreams().inNamespace(namespace).withName("slave-jenkins").get().getSpec().getTags().get(0).getName());
+      log("Second Tag is " + client.imageStreams().inNamespace(namespace).withName("slave-jenkins").get().getSpec().getTags().get(1).getName());
+
+      client.imageStreams().inNamespace(namespace).withName("slave-jenkins").delete();
```
Comment (hrishin) [inline kubernetes-examples/src/main/java/io/fabric8/openshift/examples/ImageStreamExample.java]: @piyush1594 I think we should test `client.imageStreamTags()` not client.imageStreams()

Comment (piyush1594) [reply kubernetes-examples/src/main/java/io/fabric8/openshift/examples/ImageStreamExample.java]: Yaa. just added that, means fixed the example that was not working. WIP

**Comment by piyush1594** [comment]
ok to test

**Comment by piyush1594** [comment]
@oscerd All set to go. Thanks

**Comment by piyush1594** [comment]
ok to test

**Comment by oscerd** [comment]
[merge]

**Pull request merged**